### PR TITLE
Second part of coin example.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by leonardoalt — DISMISSED** [review]

File: docs/introduction-to-smart-contracts.rst [hunk 1]
```diff
@@ -180,18 +180,26 @@ the user interface.
...
+causes all changes to be reverted if its argument evaluates to false.
+The second call to require ensures that there will not be too many coins,
+which could cause overflow errors later.
+
+On the other hand, ``send`` can be used by anyone (who already
+has some of these coins) to send coins to anyone else. If you do not have
+enough coins to send, the ``require`` call will fail and also provide the
+user with an appropriate error message string.
+
+Note that if you use
```
Comment (ChrisChinchilla) [inline docs/introduction-to-smart-contracts.rst]: Maybe make this a proper RST / Sphinx note.

File: docs/introduction-to-smart-contracts.rst [hunk 2]
```diff
@@ -180,18 +180,26 @@ the user interface.
...
 always the address where the current (external) function call came from.
 
 Finally, the functions that will actually end up with the contract and can be called
 by users and contracts alike are ``mint`` and ``send``.
 If ``mint`` is called by anyone except the account that created the contract,
-nothing will happen. On the other hand, ``send`` can be used by anyone (who already
-has some of these coins) to send coins to anyone else. Note that if you use
+nothing will happen. This is ensured by the special function ``require`` which
+causes all changes to be reverted if its argument evaluates to false.
+The second call to require ensures that there will not be too many coins,
```
Comment (ChrisChinchilla) [inline docs/introduction-to-smart-contracts.rst]: For clarity might be good to code format the 'require' here.

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
Some very small suggestions.

**Comment by chriseth** [comment]
Updated.

File: docs/introduction-to-smart-contracts.rst [hunk 3]
```diff
@@ -180,23 +180,34 @@ the user interface.
...
+user with an appropriate error message string.
+
+.. note::
+    If you use
+    this contract to send coins to an address, you will not see anything when you
+    look at that address on a blockchain explorer, because the fact that you sent
+    coins and the changed balances are only stored in the data storage of this
+    particular coin contract. By the use of events it is relatively easy to create
+    a "blockchain explorer" that tracks transactions and balances of your new coin,
+    but you have to inspect the coin contract address and not the addressess of the
```
Comment (ekpyron) [inline docs/introduction-to-smart-contracts.rst]: Typo ``addressess`` - found by spell checker.

**Review by ekpyron — APPROVED** [review]

**Pull request merged**